src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs

```diff
@@ -501,22 +501,26 @@ public bool ValueTextEquals(ReadOnlySpan<char> text)
                 // Cannot create a span directly since it gets passed to instance methods on a ref struct.
                 unsafe
                 {
-                    byte* ptr = stackalloc byte[length];
+                    byte* ptr = stackalloc byte[JsonConstants.StackallocThreshold];
                     otherUtf8Text = new Span<byte>(ptr, length);
```

Comment:
Was unsure if I needed to use the length or stackalloc threshold size for instantiating the span